server.go

```diff
@@ -514,6 +514,10 @@ func (srv *Server) serveUDP(l *net.UDPConn) error {
 			}
 			return err
 		}
+		const msgHeaderLen = 12
```

Comment:
I'll leave it up to @miekg but there's a few spots where we just hardcode 12 with a comment (like `compressedLen`). So not sure if we should just hardcode it in the `if` and add a comment above the if instead of defining this const.

Reply:
This const is only scoped to the for loop - so it doesn't leak anywhere - and it's definitely what I would consider idiomatic go. It is very much up to @miekg though. My vote is squarely for the const, but a comment would still convey the same information (albeit with slightly less clarity).

Reply:
there is a const headerSize

Reply:
@miekg I went looking for it, but obviously I didn’t look hard enough. Done.